sdks/python/apache_beam/io/gcp/pubsub_test.py

```diff
@@ -82,25 +83,37 @@ def test_proto_conversion(self):
     self.assertEqual(m_converted.attributes, attributes)
 
   def test_eq(self):
-    a = PubsubMessage(b'abc', {1: 2, 3: 4})
-    b = PubsubMessage(b'abc', {1: 2, 3: 4})
-    c = PubsubMessage(b'abc', {1: 2})
+    publish_time_secs = 1520861821
```

Comment:
Would it be simpler to reduce these 3 lines to a single line as:

```
    publish_time = Timestamp(seconds=1520861821,
                             nanos=234567000)
```

Reply:
Yes no problem; I'd copied and pasted this from another test where the seconds and nanos were used seperately; but it's not needed here.